packages/contracts-bedrock/src/universal/CrossDomainMessenger.sol

```diff
@@ -288,6 +288,12 @@ abstract contract CrossDomainMessenger is
         xDomainMsgSender = Constants.DEFAULT_L2_SENDER;
 
         if (success) {
+            // This check is identical to one above, but it ensures that the same message cannot be relayed
+            // twice, and adds a layer of protection against rentrancy.
+            require(
```

Comment:
I don't see any expect reverts for this case

Reply:
Addressed here: https://github.com/ethereum-optimism/optimism/pull/8864/files#r1443310855